src/modules/logger/logged_topics.cpp

```diff
@@ -54,6 +54,7 @@ void LoggedTopics::add_default_topics()
 	add_optional_topic("actuator_controls_status_0", 300);
 	add_topic("airspeed", 1000);
 	add_optional_topic("airspeed_validated", 200);
+	add_topic("airspeed_smoothed", 200);
```

Comment:
PX4 calls low pass filtered values "filtered", should use that convention

Reply:
See e.g. https://docs.px4.io/main/en/msg_docs/BatteryStatus.html and https://docs.px4.io/main/en/msg_docs/TecsStatus.html

Reply:
Agree, should follow px4 convention, will update.